recipes/rpy2/build.sh

```diff
@@ -10,7 +10,6 @@ if [ "$(uname)" == "Darwin" ]; then
     export CFLAGS="${CFLAGS} -mmacosx-version-min=${MACOSX_VERSION_MIN}"
     export CXXFLAGS="${CXXFLAGS} -mmacosx-version-min=${MACOSX_VERSION_MIN}"
     export CXXFLAGS="${CXXFLAGS} -stdlib=libc++"
-    export LDFLAGS="${LDFLAGS} -headerpad_max_install_names"
```

Comment:
I tested this morning for `nco` and it is still needed there. But having that in the `toolchain`, and breaking packages like this one, is just another case for we mentioned in the last meeting.